contracts/cw3-fixed-multisig/Cargo.toml

```diff
@@ -21,6 +21,7 @@ library = []
 utils = { path = "../../packages/utils", version = "0.10.3" }
 cw2 = { path = "../../packages/cw2", version = "0.10.3" }
 cw3 = { path = "../../packages/cw3", version = "0.10.3" }
+cw3-flex-multisig = { path = "../cw3-flex-multisig", version = "0.10.3" }
```

Comment:
Not a big deal, but it would be good to do it the other way around, i.e. cw3-flex importing stuff from cw3-fixed.

Reply:
Actually this is because of threshold, so it won't be needed after https://github.com/CosmWasm/cw-plus/pull/590

Reply:
I moved common implementation of proposals, ballots etc. from flex to fixed multisig.
Now is how you pointed - flex imports those from fixed.